tests/jballerina-unit-test/src/test/resources/testng.xml

```diff
@@ -300,20 +300,20 @@
             </class>
             <class name="org.ballerinalang.test.javainterop.basic.InstanceMethodTest">
```

Comment:
Since we removed all the excluded methods, We can remove this class right?

Reply:
Yes. Removed